tools/state-viewer/src/congestion_control.rs

```diff
@@ -189,6 +191,7 @@ impl PrepareBenchmarkCmd {
 
         for _ in 0..self.receipt_count {
             let receipt = self.create_receipt();
+            let receipt = ReceiptOrStateStoredReceipt::Receipt(receipt);
             queue.push(&mut trie_update, &receipt).unwrap();
         }
 
```

Comment:
Would be nice to have an integration test which goes through the protocol upgrade. Not sure how to achieve that though.

Reply:
I think we have that for congestion control but let me double check it's doing what you're suggesting.

Reply:
We do have that test but it doesn't cover this case. I'll either improve it a implement a new one tomorrow.